Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-AKS-cluster/aks-cluster.tf

```diff
@@ -60,11 +60,9 @@ module "aks" {
   vnet_subnet_id      = data.terraform_remote_state.existing-lz.outputs.aks_subnet_id
   mi_aks_cp_id        = azurerm_user_assigned_identity.mi-aks-cp.id
   la_id               = azurerm_log_analytics_workspace.aks.id
-  # gateway_name        = data.terraform_remote_state.existing-lz.outputs.gateway_name
-  # gateway_id          = data.terraform_remote_state.existing-lz.outputs.gateway_id
   private_dns_zone_id = azurerm_private_dns_zone.aks-dns.id
-  wnp_count = var.wnp_count
-  appgwSubnet_Id = data.terraform_remote_state.existing-lz.outputs.appgwSubnet_Id
+  wnp_count           = var.wnp_count
+  appdevs             = data.azuread_group.appdevs
```

Comment:
I kept the same naming under 03-AAD-create. The variable is needed because I set the aks admins in terraform. I just found something I need to fix so how about I call it aks_admin_group so its less confusing.